Xlib/protocol/display.py

```diff
@@ -532,7 +532,7 @@ def send_and_recv(self, flush = None, event = None, request = None, recv = None)
             if ws:
                 try:
                     i = self.socket.send(self.data_send)
-                except socket.error, err:
+                except (socket.error, err) as err:
```

Comment:
2to3 fixer suggests to make it `except socket.error as err:`. Comma doesn't mean a list in Python 2 `except` statement.